### PR TITLE
PLAT-54096: Add unit test for DatePicker, ExpandablePicker, TimePicker, internal/Picker

### PR DESCRIPTION
**Pull request by Baekwoo-s**
### Issue Resolved / Feature Added
Added a11y unit test of DatePicker, ExpandablePicker, TimePicker, internal/Picker

### Links
PLAT-54096

### Comments
Enact-DCO-1.0-Signed-off-by: Baekwoo Jung (baekwoo.jung@lge.com)

### CODE REVIEW

File: packages/moonstone/DatePicker/tests/DatePicker-specs.js [hunk 1]
```diff
@@ -85,4 +85,88 @@ describe('DatePicker', () => {
 
 		expect(actual).to.equal(expected);
 	});
+
+	it('should set "dayAriaLabel" to day picker', function () {
+		const label = 'custom day aria-label';
+		const subject = mount(
+			<DatePicker dayAriaLabel={label} title="Date" value={new Date(2000, 0, 1)} open />
```
Comment (ybsung) [inline packages/moonstone/DatePicker/tests/DatePicker-specs.js]: Please define `open` before `title` and `value`.

File: packages/moonstone/DatePicker/tests/DatePicker-specs.js [hunk 2]
```diff
@@ -85,4 +85,88 @@ describe('DatePicker', () => {
...
+		const expected = label;
+		const actual = dayPicker.prop('aria-label');
+
+		expect(actual).to.equal(expected);
+	});
+
+	it('should set "monthAriaLabel" to month picker', function () {
+		const label = 'custom month aria-label';
+		const subject = mount(
+			<DatePicker monthAriaLabel={label} title="Date" value={new Date(2000, 0, 1)} open />
```
Comment (ybsung) [inline packages/moonstone/DatePicker/tests/DatePicker-specs.js]: same above

File: packages/moonstone/ExpandablePicker/tests/ExpandablePicker-specs.js [hunk 3]
```diff
@@ -62,4 +62,68 @@ describe('ExpandablePicker Specs', () => {
 
 		expect(actual).to.equal(expected);
 	});
+
+	it('should set "checkButtonAriaLabel" to check button', function () {
+		const label = 'custom check button aria-label';
+		const expandablePicker = mount(
+			<ExpandablePickerBase checkButtonAriaLabel={label} title="Options" open>
```
Comment (ybsung) [inline packages/moonstone/ExpandablePicker/tests/ExpandablePicker-specs.js]: same above

**Comment by Baekwoo-s** [comment]
I applied all the above comments.

File: packages/moonstone/DatePicker/tests/DatePicker-specs.js [hunk 4]
```diff
@@ -71,7 +71,7 @@ describe('DatePicker', () => {
 
 	it('should accept an updated JavaScript Date for its value prop', function () {
 		const subject = mount(
-			<DatePicker title="Date" value={new Date(2000, 0, 1)} open />
+			<DatePicker title="Date" open value={new Date(2000, 0, 1)} />
```
Comment (ybsung) [inline packages/moonstone/DatePicker/tests/DatePicker-specs.js]: Please order open, title, and value.

**Comment by ybsung** [comment]
LGTM

**Pull request merged**